zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/IdentitySetupInitializeProcessor.java

```diff
@@ -239,21 +237,20 @@ final var record =
   private void addAllPermissions(final long roleKey) {
 
     for (final AuthorizationResourceType resourceType : AuthorizationResourceType.values()) {
+      if (resourceType == AuthorizationResourceType.UNSPECIFIED) {
+        // We shouldn't add unspecified resource type
+        continue;
+      }
       // TODO: refactor when Roles use String IDs as unique identifiers
       final var record =
           new AuthorizationRecord()
-              .setOwnerKey(roleKey)
               .setOwnerId(String.valueOf(roleKey))
-              .setOwnerType(AuthorizationOwnerType.ROLE);
-      record.setResourceType(resourceType);
-
-      for (final PermissionType permissionType : resourceType.getSupportedPermissionTypes()) {
-        final var permission =
-            new Permission().setPermissionType(permissionType).addResourceId(WILDCARD_PERMISSION);
-        record.addPermission(permission);
-      }
+              .setOwnerType(AuthorizationOwnerType.ROLE)
```

Comment:
❓ Did you leave out the authorizationKey because we will remove it later?

Reply:
Yep, I saw the comment yesterday 👍